app/helpers/Intersect.js

```diff
@@ -402,3 +432,21 @@ export function disableDebugMode() {
   // Kick off intersection calculations to restore data
   checkIntersect();
 }
+
+/**
+ * Records where user has crossed paths to avoid repeated notifications or checking
+ *
+ * @param {array} localArray - Array of user locations
+ * @param {number} i - Index of current location in the localArray
+ * @param {array} infectedLocation - Array of infected locations
+ * @param {number} newIntersections - Counter of new intersections found
+ */
+function addNewIntersection(localArray, i, infectedLocation, newIntersections) {
+  console.log('New intersection');
+
+  if ('crossedPaths' in localArray[i])
+    localArray[i]['crossedPaths'].push(infectedLocation);
+  else localArray[i]['crossedPaths'] = [infectedLocation];
+
+  return newIntersections + 1;
```

Comment:
This method shouldn't know about the count. It always adds 1, so the count is irrelevant. You could leave count in the parent method.